### ISSUE TITLE
Bring CSS styles from Read the Docs addons into ad client

### ISSUE THREAD

**Issue by humitos** [readthedocs/ethical-ad-client#193]
Read the Docs publisher define specific CSS rules to style the ad in a different way. We talked about moving these styles into the ad client itself and rely on light / dark styles _only_ instead of theme based like `ethical-alabaster`.

Reference:
* https://github.com/readthedocs/addons/pull/309#pullrequestreview-2064411008
* https://github.com/readthedocs/addons/pull/295#discussion_r1605555978

**Comment by ericholscher** [comment]
I put this in Q2, because it's a nice refactor while we're doing this work.

**Comment by davidfischer** [comment]
Are we thinking of bringing styles from RTD to the ad client? Or are we thinking about having RTD just use the built-in styles/ad types rather than doing something custom?

**Comment by humitos** [comment]
Yeah, I think we are talking about the later: "use dark/light styles from the EA client in Read the Docs publisher".

_However_, I think we may still want some extra CSS classes in for some themes/doctools. I'm mentioning this because I found, for example, Pelican that renders in a weird way:

![Peek 2024-05-24 17-10](https://github.com/readthedocs/ethical-ad-client/assets/244656/0989ffdc-ec9f-4e3a-afc4-7fa57773c2c6)
